lib/transport/reusable_data/reusable_data.ex

```diff
@@ -167,8 +168,10 @@ defmodule Transport.ReusableData do
     |> Datasets.get(dataset.slug)
     |> case do
       {:ok, %{"id" => id}} -> id
-      {:ok, _}            -> nil
-      {:error, _}          -> nil
+      {:ok, _}             -> nil
+      {:error, error}          ->
```

Comment:
```->``` are not aligned anymore, so either do not add more than one space, or align them.